app/routes/routes.js

```diff
@@ -83,7 +84,22 @@ module.exports = (app, models) => {
                         .json(temp);
                 }
                 else {
-                    return res.json({token: jwt.sign({ email: user.email, id: user.id, type: req.body.type }, 'ClientVendor')});
+                
+                        var Verify = models.verification;
+
+                        Verify.findOne({where: {id : user.id}}).then(function(verify) {
+                        
+                                temp.status = 200;
+                                console.log(verify);
+                                if (!verify.emailverified) {
```

Comment:
Can we use 403 as the status code for Not Verified User?

Reply:
Yes we can add that